docs/fixes/2026-08-19-asciicast-runsession-windows-ci-timeout.md

```
@@ -0,0 +1,88 @@
+# Fix: `TestRunSessionExecutesScriptedShellActions`/`TestRunSessionAppliesDirectoryAndEnvironment` timeout on Windows CI
+
+**Date:** 2026-08-19
+
+## Summary
+
+`Acceptance Tests (windows, shard 4/10)` failed with:
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)